### DISCUSSION TITLE
Data Missing on App Once Sync Occurs

### DISCUSSION THREAD

**Discussion by alexhogan18** [opengisch/QField#2796] [Q&A]
I am using the qfield application on an android device, I will add a point, go push changes, then synchronise (so hopefully other users can see the changes immediately), the data point is now missing when viewing on the app, however when viewing the project in google chrome the change can be seen and say applied, I can also see the new point when I sync the project in qgis, any help would be appreciated

**Comment by suricactus** [comment]
From your question I assume you are expecting that after pressing "synchronize" it will automatically sync all devices that have this project. However, the syncrhonize button will make sure the applied changes are downloaded only on your device.

If there is another user that has the same project on their device, they also need to press synchronize, so they can obtain the latest copy of the project on their phone.

Hope it answers your question.

**Comment by alexhogan18** [comment]
No my issue is that I push the changes into the cloud, then I syncronise the project and the data I just added into the project disapears, this new data is still in the project on the cloud as I can see it in the changes and it has the applied status, however i cannot see this data on my tablet now, after syncing

**Reply by suricactus** [reply]
Ok clear. Can you share your project name or upload your project here, so we can check what and if there is an issue?

**Reply by alexhogan18** [reply]
RMPFootpathInspections__QFieldCloud_ let me know if you can view this

**Reply by alexhogan18** [reply]
@suricactus I think this may be an issue with the new version of the app

**Reply by suricactus** [reply]
I will have a look about your issue in the coming weeks about your issue.

**Reply by alexhogan18** [reply]
@suricactus is there anyway I can use an older version of the app?

**Reply by alexhogan18** [reply]
@suricactus I have noticed that this only occurs when there are multiple tablets with the same project open at once

**Comment by suricactus** [comment]
@alexhogan18 we failed to reproduce your issue on neither of two different devices using the latest QField. Do you experience this problem every time or just sometimes?
